source_gen/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-## 1.2.3
+## 1.2.3-dev
 
-* Require `analyzer: '>=4.3.0 <5.0.0'`
+* Require `analyzer: ^4.4.0`
```

Comment:
I'm surprised that you can get away with `4.4.0`, not `4.6.0`, where we added `DartType.element2`, so not just to `InterfaceType`.

Reply:
Ah, I probably need to bump it further then.